vars/parseStageInfo.groovy

```diff
@@ -344,7 +344,13 @@ Map call(Map config = [:]) {
         // Assemble the stage test tags - add the cluster size to each tag group
         result['test_tag'] = ''
         for (group in tag.split(' ')) {
-            result['test_tag'] += group + ',' + cluster_size + ' '
+            if (group == '+') {
+                // The '+' is a special string in the tag field for defining multiple groups of tags
+                result['test_tag'] += group + ' '
+            }
+            else {
+                result['test_tag'] += group + ',' + cluster_size + ' '
```

Comment:
FWIW, this could have been done with:

```suggestion
            result['test_tag'] += group + (group != '+' : ',' + cluster_size ? '') + ' '
```

I think.

Reply:
The ':' and '?' where flipped, but it does work, thanks:
```
result['test_tag'] += group + (group != '+' ? ',' + cluster_size : '') + ' '
```

Reply:
Oops. yes.  Sorry.